packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -279,7 +279,7 @@ export class ProductAttributeValue extends PureComponent {
               block="ProductAttributeValue"
               elem="SubLabel"
             >
-                { `(${subLabel})` }
+                { subLabel ? `(${subLabel})` : undefined }
```

Comment:
isn't it fixed in this PR?
https://github.com/scandipwa/scandipwa/pull/3838/files